notebooks/official/feature_store/online_feature_serving_and_vector_retrieval_bigquery_data_with_feature_store.ipynb

```diff
@@ -88,20 +88,6 @@
         "- Use the online server to search nearest neighbors."
```

Comment:
<p>per offline chat, please double confirm with Quanjie on whether to use <code>data_key</code> or <code>id</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>